### PR TITLE
Create jonah-andersson.md

### PR DESCRIPTION
**Pull request by jonahandersson**
Contribution profile to https://www.womenofdot.net :)

### CODE REVIEW

File: content/profiles/jonah-andersson.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+github: "https://github.com/jonahandersson"
+twitch: ""
+instagram: "https://instagram.com/jonahandersson_ladyprogrammer"
+blogsite: "https://jonahandersson.tech/"
+linkedin: "https://www.linkedin.com/in/jonahandersson"
+comments: false
+summary: "I am a Microsoft MVP for Azure, Software Engineer, Certified Azure Developer, Microsoft Certified Trainer
+Tech Mentor, Public Speaker, Author, Blogger. Genuine, Humble, Brave, Sincere. Love Learning! 
+"
+draft: true
```
Comment (Layla-P) [inline content/profiles/jonah-andersson.md:16]: Hey Jonah!
This line needs to be deleted for the profile to go live :-)

**Review by Layla-P — CHANGES_REQUESTED** [review]
Just a quick change and then we are good to go!

File: content/profiles/jonah-andersson.md (hunk 1 above)
Comment (Layla-P) [reply content/profiles/jonah-andersson.md]: I did it for you ;-)

Comment (jonahandersson) [reply content/profiles/jonah-andersson.md]: Thank you @Layla-P for updating and editing it.

**Pull request merged**